### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by yosinv**
#607 edate is set for TODAY in the start menu

(app.js,)

### CODE REVIEW

**Comment by danielhers** [comment]
Hi Yosi, please squash your commits to one commit (see contribution guidelines).

File: static/js/app.js [hunk 1]
```diff
@@ -73,8 +73,9 @@ $(function () {
             this.controlmeasure = 0;
             this.district = 0;
             this.case_type = 0;
+            this.edate = new Date();
 
-            this.dateRanges = [new Date($("#sdate").val()), new Date($("#edate").val())];
+            this.dateRanges = [new Date($("#sdate").val()), new Date(this.edate)];
```
Comment (danielhers) [inline static/js/app.js]: The `#edate` control should also be updated to show `this.edate`, no?

**Comment by yosinv** [comment]
how do I squash all the commit to one commit?

**Comment by danielhers** [comment]
http://stackoverflow.com/questions/5189560/squash-my-last-x-commits-together-using-git

**Pull request closed without merging**